fv3core/fv3core/stencils/fv_subgridz.py

```diff
@@ -838,15 +843,19 @@ def make_storage():
         self._tmp_cpm = make_storage()
         self._ratios = {0: 0.25, 1: 0.5, 2: 0.999}
 
-    def __call__(self, state: Mapping[str, pace.util.Quantity], timestep: float):
+    def __call__(
+        self, state: DycoreState, u_dt: Quantity, v_dt: Quantity, timestep: float
+    ):
         """
         Performs dry convective adjustment mixing on the subgrid vertical scale.
         Args:
             state: see arg_specs, includes mainly windspeed, temperature,
-                   pressure and tracer variables
+                   pressure and tracer variables that are in the DycoreState
+            u_dt: x-wind tendency for the dry convective windspeed adjustment
+            v_dt: y-wind tendency for the dry convective windspeed adjustment
             timestep:  time to progress forward in seconds
         """
-        if state.pe[self._is, self._js, 0] < 2.0:
+        if state.pe.data[self._is, self._js, 0] < 2.0:
```

Comment:
oh previously we only ever passed storages to fv_subgridz, because it was only run as part of the test... Now we are hooking it into Driver, it actually gets quantities